tests/ert3/storage/test_storage.py

```diff
@@ -27,6 +27,7 @@ def test_ensemble_size_zero(tmpdir, ert_storage):
             experiment_name="my_experiment",
             parameters=[],
             ensemble_size=0,
+            responses=[],
```

Comment:
I see that none of the storage tests are passing responses to an experiment. I think we should add at least one test doing that! Also, is there no behavior we can detect based on the content of responses? What happens if a record name is both a parameter and a response for instance?